src/meta/proto-conv/tests/it/proto_conv.rs

```diff
@@ -268,37 +316,60 @@ fn test_load_old() -> anyhow::Result<()> {
 
     // DatabaseMeta is loadable
     {
-        let db_meta_v1: Vec<u8> = vec![
-            34, 10, 10, 3, 120, 121, 122, 18, 3, 102, 111, 111, 42, 2, 52, 52, 50, 10, 10, 3, 97,
-            98, 99, 18, 3, 100, 101, 102, 162, 1, 23, 50, 48, 49, 52, 45, 49, 49, 45, 50, 56, 32,
-            49, 50, 58, 48, 48, 58, 48, 57, 32, 85, 84, 67, 170, 1, 23, 50, 48, 49, 52, 45, 49, 49,
-            45, 50, 57, 32, 49, 50, 58, 48, 48, 58, 48, 57, 32, 85, 84, 67, 178, 1, 7, 102, 111,
-            111, 32, 98, 97, 114, 160, 6, 2, 168, 6, 1,
-        ];
-
-        let p: pb::DatabaseMeta =
-            common_protos::prost::Message::decode(db_meta_v1.as_slice()).map_err(print_err)?;
-
-        let got = mt::DatabaseMeta::from_pb(p).map_err(print_err)?;
-
-        let want = new_db_meta_v1();
-        assert_eq!(want, got);
-
-        let db_meta_v2: Vec<u8> = vec![
-            34, 10, 10, 3, 120, 121, 122, 18, 3, 102, 111, 111, 42, 2, 52, 52, 50, 10, 10, 3, 97,
-            98, 99, 18, 3, 100, 101, 102, 162, 1, 23, 50, 48, 49, 52, 45, 49, 49, 45, 50, 56, 32,
-            49, 50, 58, 48, 48, 58, 48, 57, 32, 85, 84, 67, 170, 1, 23, 50, 48, 49, 52, 45, 49, 49,
-            45, 50, 57, 32, 49, 50, 58, 48, 48, 58, 48, 57, 32, 85, 84, 67, 178, 1, 7, 102, 111,
-            111, 32, 98, 97, 114, 194, 1, 1, 1, 160, 6, 2, 168, 6, 1,
-        ];
-
-        let p: pb::DatabaseMeta =
-            common_protos::prost::Message::decode(db_meta_v2.as_slice()).map_err(print_err)?;
-
-        let got = mt::DatabaseMeta::from_pb(p).map_err(print_err)?;
-
-        let want = new_db_meta();
-        assert_eq!(want, got);
+        {
+            let db_meta_v1: Vec<u8> = vec![
+                34, 10, 10, 3, 120, 121, 122, 18, 3, 102, 111, 111, 42, 2, 52, 52, 50, 10, 10, 3,
+                97, 98, 99, 18, 3, 100, 101, 102, 162, 1, 23, 50, 48, 49, 52, 45, 49, 49, 45, 50,
+                56, 32, 49, 50, 58, 48, 48, 58, 48, 57, 32, 85, 84, 67, 170, 1, 23, 50, 48, 49, 52,
+                45, 49, 49, 45, 50, 57, 32, 49, 50, 58, 48, 48, 58, 48, 57, 32, 85, 84, 67, 178, 1,
+                7, 102, 111, 111, 32, 98, 97, 114, 160, 6, 2, 168, 6, 1,
+            ];
+
+            let p: pb::DatabaseMeta =
+                common_protos::prost::Message::decode(db_meta_v1.as_slice()).map_err(print_err)?;
+
+            let got = mt::DatabaseMeta::from_pb(p).map_err(print_err)?;
+
+            let want = new_db_meta_v1();
+            assert_eq!(want, got);
```

Comment:
Is the `v1` message changed?

The serialized old message should not be modified to assert backward compatibility.

Reply:
`v1` message has not been changed, `diff` make some mistakes.

Reply:
I think it's because of the additional `{` at line 320.

Removing it will get rid of this reformatting.